rust/src/lib/ifaces/ovs.rs

```diff
@@ -245,6 +247,14 @@ impl OvsBridgeInterface {
 #[serde(rename_all = "kebab-case", deny_unknown_fields)]
 #[non_exhaustive]
 pub struct OvsBridgeConfig {
+    #[serde(skip_serializing, default)]
+    /// Only validate for applying, when set to `true`, extra OVS patch ports
+    /// will not fail the verification. Default is false.
+    /// This property will not be persisted, every time you modify
+    /// ports of specified OVS bridge, you need to explicitly define this
+    /// property if not using default value.
+    /// Deserialize from `allow-extra-patch-ports`.
```

Comment:
My main problem with this property is that it goes against the "declarative" idea of Nmstate. It describes "how" not the "what". The user should not care about this implementation detail. Othet than that, the patch is doing what it is being described.

Reply:
Right, I have been thinking about this. IMHO it is not declarative because it is specifying "how" to do it and not "what" do you want to configure. Having said this, as it is not persistent and therefore not reported on "show" so it won't affect the "show" -> "apply" workflow, I am fine with this change.

Reply:
We thought about this, and it was really hard to come up w/ a declarative way of expressing this intent. 

Can you suggest one @ffmancera ? We've failed at that exercise.